bower.json

```diff
@@ -2,30 +2,26 @@
   "name": "vagrant",
   "version": "0.0.0",
   "dependencies": {
-    "angular": "^1.3.0",
+    "angular": "1.4.1",
     "bootstrap-sass-official": "^3.2.0",
-    "angular-animate": "^1.3.0",
-    "angular-aria": "^1.3.0",
-    "angular-cookies": "~1.3.15",
-    "angular-messages": "^1.3.0",
-    "angular-resource": "^1.3.0",
-    "angular-route": "^1.3.0",
```

Comment:
We have switched from ngRoute to ui-router, so this dependency is not needed anymore.